docs/ref/databases.txt

```diff
@@ -31,14 +31,14 @@ Persistent connections
 ----------------------
 
 Persistent connections avoid the overhead of reestablishing a connection to
-the database in each request. They're controlled by the
-:setting:`CONN_MAX_AGE` parameter which defines the maximum lifetime of a
-connection. It can be set independently for each database.
+the database in each :class:`~django.http.HttpRequest` request. They're
+controlled by the :setting:`CONN_MAX_AGE` parameter which defines the maximum
+lifetime of a connection. It can be set independently for each database.
 
 The default value is ``0``, preserving the historical behavior of closing the
-database connection at the end of each request. To enable persistent
-connections, set :setting:`CONN_MAX_AGE` to a positive integer of seconds. For
-unlimited persistent connections, set it to ``None``.
+database connection at the end of each :class:`~django.http.HttpRequest`. To
+enable persistent connections, set :setting:`CONN_MAX_AGE` to a positive
+integer of seconds. For unlimited persistent connections, set it to ``None``.
```

Comment:
I would undo all these changes and just have the first reference to `request` be `HTTP request`.